src/core/theme/radius.ts

```diff
@@ -1,4 +1,4 @@
 export const radius = {
   basic: '2px',
-  focus: '4px',
+  focus: '2px',
```

Comment:
I think the 4px radius is closer to the current design if that's a viable value otherwise.